Source/WebCore/platform/GStreamer.cmake

```diff
@@ -1,6 +1,11 @@
 include(CheckCXXSymbolExists)
 
 if (ENABLE_VIDEO OR ENABLE_WEB_AUDIO)
+    find_package(GLib 2.70.0 REQUIRED COMPONENTS GioUnix)
```

Comment:
This line shouldn't be needed because both `Source/cmake/Options{GTK,WPE}.cmake` already include `GioUnix` in the list of components they check. Adding the library to the `WebCore` target should be enough.